### PR TITLE
WIP: add archlinux support

### PR DESCRIPTION
**Pull request by SantiagoTorres**
This patch adds the required commands to support archlinux/pacman based
containers.

### CODE REVIEW

**Comment by SantiagoTorres** [comment]
Hi @nishakm ! I owed you this. I'm assuming you'd like some tests for this feature? :+1: 

Also, I think that DCA bot has a typo, the commit was signed, yet it didn't have a signed-off-by (-S vs -s). I ammended the commit to include the line, but I wonder if that check is intentional.

**Comment by SantiagoTorres** [comment]
> Thank you for this! I'll take a look at what the DCO bot did. Do you have an image I can test this on?

Yes, we have an ArchLinux image under archlinux/base on [DockerHub](https://hub.docker.com/r/archlinux/base). It's a little big, but once we trim the fat we'll have it on [the official library](https://github.com/docker-library/official-images/pull/5459)

File: src/command_lib/snippets.yml [hunk 1]
```diff
@@ -90,3 +90,31 @@ apk:
             container:
               - 'apk -R info python 2>/dev/null | tail -n +2 | head -n -1 | cut -f2 -d":" | cut -f1 -d"."'
         delimiter: "\n"
+
+
+pacman:
+  install: '-Syu'
+  remove: '-Rcs'
+  packages:
```
[the remark this answers is not in the document]
Comment (nishakm) [reply src/command_lib/snippets.yml]:
```
pacman:
  install: '-Syu'
  remove: '-Rcs'
  packages: 'pacman'
```

[the remark this answers is not in the document]
Comment (SantiagoTorres) [reply src/command_lib/snippets.yml]: Ah yeah I was a little confused about this but I added it anyway. Should I remove it?

[the remark this answers is not in the document]
Comment (nishakm) [reply src/command_lib/snippets.yml]: No, just replace all the stuff under 'packages' with 'pacman' in this file.

**Comment by nishakm** [comment]
@SantiagoTorres Works great on archlinux/base! I'd like to merge this. Would you mind squashing the two commits and resubmitting without the WIP?
I'm not sure why the DCO bot placed the required on your previous commit as your original commit is no longer available. Do you want to experiment and resubmit this PR on a new branch with your original commit message if you remember what that looked like? If not, then I'll ask the maintainer of the DCO bot if they have logs or something. Thanks so much!

**Comment by SantiagoTorres** [comment]
> @SantiagoTorres Works great on archlinux/base! I'd like to merge this. Would you mind squashing the two commits and resubmitting without the WIP?

Sure thing! Let me re-do this PR Then

> I'm not sure why the DCO bot placed the required on your previous commit as your original commit is no longer available. Do you want to experiment and resubmit this PR on a new branch with your original commit message if you remember what that looked like?

I don't think it's worth the hassle imho. Thanks!

**Comment by nishakm** [comment]
Refer to #189 for follow-up PR. Thanks again @SantiagoTorres!

**Pull request closed without merging**